python/compiler/aiecc/main.py

```diff
@@ -445,7 +445,10 @@ async def chesshack(self, task, llvmir, aie_target):
         await self.do_call(
             task,
             [
-                "chess-llvm-link",
+                # The path below is cheating a bit since it refers directly to the AIE1
+                # version of llvm-link, rather than calling the architecture-specific
+                # tool version.
+                opts.aietools_path + "/tps/lnx64/target/bin/LNa64bin/chess-llvm-link",
```

Comment:
So it does not matter which version is used?

Reply:
The different synopsys versions are currently pretty close to one another, and 'HEAD' is diverging relatively quickly...  This is at some level kicking the can down the road, but it's better than what we have.